src/main/java/com/springbootWithJPA/springBootRestExample/Employee.java

```diff
@@ -1,56 +1,59 @@
-package com.springbootWithJPA.springBootRestExample;
-
-import javax.persistence.*;
+package com.springbootwithjpa.springbootrestexample;
 
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.persistence.Id;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
 @Entity
 @Table(name = "employees")
 public class Employee {
-        @Id
-        @GeneratedValue(strategy = GenerationType.AUTO)
-        private int id;
-        private String firstName;
-        private String lastName;
-        private String emailId;
-
-        public int getId() {
-                return id;
-        }
-
-        public void setId(int id) {
-                this.id = id;
-        }
-
-        public String getFirstName() {
-                return firstName;
-        }
-
-        public void setFirstName(String firstName) {
-                this.firstName = firstName;
-        }
-
-        public String getLastName() {
-                return lastName;
-        }
-
-        public void setLastName(String lastName) {
-                this.lastName = lastName;
-        }
-
-        public String getEmailId() {
-                return emailId;
-        }
-
-        public void setEmailId(String emailId) {
-                this.emailId = emailId;
-        }
-
-        @Override
-        public String toString() {
-                return "Employee{" +
-                        "id=" + id +
-                        ", firstName='" + firstName + '\'' +
-                        ", lastName='" + lastName + '\'' +
-                        ", emailId='" + emailId + '\'' +
-                        '}';
-        }
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private int id;
+    private String firstName;
+    private String lastName;
+    private String emailId;
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
```

Comment:
Use lombok to remove getters and setters

Reply:
addressed under https://github.com/Jashika/springBootRestExample/commit/ace916e5da0effe859f67b9ba3b075e0064cc3bb